src/streams.c

```diff
@@ -779,27 +779,13 @@ static Obj PRINT_OR_APPEND_TO(Obj args, int append)
 
     /* first entry is the filename                                         */
     filename = ELM_LIST(args,1);
-    while ( ! IsStringConv(filename) ) {
-        filename = ErrorReturnObj(
-            "%s: <filename> must be a string (not a %s)",
-            (Int)funcname, (Int)TNAM_OBJ(filename),
-            "you can replace <filename> via 'return <filename>;'" );
-    }
 
     /* try to open the file for output                                     */
     i = append ? OpenAppend( CSTR_STRING(filename) )
                : OpenOutput( CSTR_STRING(filename) );
     if ( ! i ) {
         if (strcmp(CSTR_STRING(filename), "*errout*") == 0) {
-            // When trying to print an error opening *errout* failed,
-            // We exit GAP after trying to print an error.
-            // First try printing an error to stderr
-            int ret = fputs("gap: panic, could not open *errout*!\n", stderr);
-            // If that failed, try printing to stdout
-            if(ret == EOF) {
-                fputs("gap: panic, could not open *errout*!\n", stdout);
-            }
-            SyExit(1);
+            Panic("gap: failed to open *errout*!\n");
```

Comment:
I agree wth using `Panic` here, but please remove the trailing `!\n`, and the leading `gap: `